src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/ManagingOfficerCorporateValidator.java

```diff
@@ -1,11 +1,14 @@
 package uk.gov.companieshouse.overseasentitiesapi.validation;
 
+import java.time.LocalDate;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
 import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerCorporateDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerIndividualDto;
```

Comment:
From sonarqube: "remove this unused import 'uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerIndividualDto'."